helpers.go

```diff
@@ -0,0 +1,29 @@
+package ssz
+
+import "github.com/prysmaticlabs/prysm/shared/hashutil"
```

Comment:
Why do you need something from Prysm? Let’s avoid this because it is a circular dep for Prysm to depend on this project

Reply:
we already have a hash function in the ssz package. Lets use that, although it needs to be changed to sha256 from keccak256

Reply:
https://github.com/prysmaticlabs/go-ssz/blob/2e84733edbac32aca6d47feafc4441e43b10047f/hash.go#L44